llvm/lib/Target/AArch64/AArch64InstrInfo.td

```diff
@@ -4020,6 +4079,33 @@ defm STNPD : StorePairNoAlloc<0b01, 1, FPR64Op, simm7s8, "stnp">;
 defm STNPQ : StorePairNoAlloc<0b10, 1, FPR128Op, simm7s16, "stnp">;
 }
 
+// Armv9.6-a Load/store no-allocate pair (FEAT_LSUI)
```

Comment:
Where does the "no-allocate" in these comments and names come from? If it's referring to what the architecture reference manual calls "non-temporal hint", then only a few of these instructions are non-temporal, so the comment doesn't apply to the rest.

Reply:
Thanks. Done